### DISCUSSION TITLE
Authorship, license, copyright, acknowledgements, and references

### DISCUSSION THREAD

**Discussion by wlandau** [openpharma/brms.mmrm#25] [General]
All, please look at the authors list, copyright/license, and acknowledgements/references in the DESCRIPTION, LICENSE, LICENSE.md, and README.Rmd files. Please let me know if I have missed anything that should be included or if I should omit or change anything mentioned. (Let's plan to cite the Novartis vignettes when they are public and a citation is available.)

cc @kkmann, @yonicd, @andrew-bean, @chstock, @danleibovitz, @weberse2

**Comment by kkmann** [comment]
I think mmrm lists companies as cph and fnd https://github.com/openpharma/mmrm/blob/main/DESCRIPTION following best practices. Apart from that, I would usually only want to be listed as a contributor after making a somewhat meaningful contribution to the code base/docs x). Up to you how you want to handle this, happy to be skipped or ctb for now.

**Comment by wlandau** [comment]
> I think mmrm lists companies as cph and fnd https://github.com/openpharma/mmrm/blob/main/DESCRIPTION following best practices. 

I have been using "cph" for companies too. Looks like tidyverse packages have also added "fnd" since last I checked. Maybe "fnd" is appropriate here too.

> Apart from that, I would usually only want to be listed as a contributor after making a somewhat meaningful contribution to the code base/docs x). Up to you how you want to handle this, happy to be skipped or ctb for now.

Contributed code is a clear measure of contributions, but I think it misses the important scoping and brainstorming discussions we have had. Your contribution seems greater than or equal to "ctb".

**Comment by chstock** [comment]
I made some edits: https://github.com/RConsortium/brms.mmrm/commit/15f42523f1b52fd393214120f628e1e40d9a1142

**Comment by wlandau** [comment]
Thanks, @chstock! I see the author list now includes @kkmann, you, @yonicd, and me. I like that we are all included.
